pageserver/src/walreceiver/connection_manager.rs

```diff
@@ -760,7 +760,7 @@ fn wal_stream_connection_string(
         })?;
     let (host, port) = utils::connstring::connection_host_port(&me_conf);
     Ok(format!(
-        "host={host} port={port} options='-c ztimelineid={timeline_id} ztenantid={tenant_id}'"
+        "host={host} port={port} options='-c timeline_id={timeline_id} tenant_id={tenant_id}'"
```

Comment:
This change needs to done at the same time with the safekeepers. Can we guarantee that when we deploy this? Maybe it'd be best to leave this as it is for now. Or change the safekeeper to accept both `ztimelineid` and 'timeline_id` now, and after some time has passed so that we're sure there are no old safekeepers running and that we won't need to revert to an older version, change all these clients.

Reply:
Thank you, I've extracted it into a [separate commit](https://github.com/neondatabase/neon/pull/2433/commits/a5fd46929a6ffc77bcfe9863f8f40dfc52f488ff).